react/components/ContentWrapper.js

```diff
@@ -30,24 +30,26 @@ class ContentWrapper extends Component {
       history,
       headerContent,
       namespace,
+      hideBackButton = false,
     } = this.props
     const { shouldShowError } = this.state
 
+    const backButtonConfigs = {
+      linkLabel: backButton
+        ? backButton.title
+          ? backButton.title
+          : intl.formatMessage({ id: backButton.titleId })
+        : intl.formatMessage({ id: 'commons.back' }),
+      onLinkClick: () => history.push(backButton ? backButton.path : '/'),
+    }
+
     return (
       <section className="vtex-account__page w-100 w-80-m">
         <header>
```

Comment:
Not related but shouldn't the `header` tag be within the `PageHeader` component?

Reply:
PageHeader is a component from Styleguide that is used here at the ContentWrapper as the header of the my-account pages!

Reply:
Good point.